src/sentry/api/endpoints/rule_snooze.py

```diff
@@ -126,3 +120,55 @@ def post(self, request: Request, project, rule_id) -> Response:
             serialize(rule_snooze, request.user, RuleSnoozeSerializer()),
             status=status.HTTP_201_CREATED,
         )
+
+    def delete(self, request: Request, project, rule_id) -> Response:
+        if not features.has("organizations:mute-alerts", project.organization, actor=request.user):
+            raise AuthenticationFailed(
+                detail="This feature is not available for this organization.",
+                code=status.HTTP_401_UNAUTHORIZED,
+            )
+
+        serializer = RuleSnoozeValidator(data=request.data)
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+        data = serializer.validated_data
+        rule = self.get_rule(rule_id)
+        user_id = request.user.id if data.get("target") == "me" else None
+
+        if not can_edit_alert_rule(rule, project.organization, user_id, request.user):
+            raise AuthenticationFailed(
+                detail="Requesting user cannot mute this rule.", code=status.HTTP_401_UNAUTHORIZED
+            )
+
+        kwargs = {self.rule_field: rule}
+
+        try:
+            rulesnooze = RuleSnooze.objects.get(
+                user_id=user_id,
+                owner_id=request.user.id,
+                until=data.get("until"),
+                **kwargs,
+            )
+        except RuleSnooze.DoesNotExist:
+            return Response(
+                {"detail": "This rulesnooze object doesn't exist."},
+                status=status.HTTP_404_NOT_FOUND,
+            )
+
+        rulesnooze.delete()
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+@region_silo_endpoint
+class RuleSnoozeEndpoint(BaseRuleSnoozeEndpoint):
+    rule_type = RuleType.ISSUE_ALERT.value
```

Comment:
Nit: Probably don't even need `rule_type` anymore

Reply:
oh yeah it's literally not being used anymore huh